main.css

```diff
@@ -477,6 +477,131 @@ body {
   border: 1px solid hsla(0, 100%, 50%, 0.48);
 }
 
+/* Modal design for modile */
+
+.modal {
```

Comment:
- [ ] [REQUIRED] Good job so far! Please implement `blur` effect for background of your modal and pay attention to your modal's `top` and `bottom` parts because they `stick` to the border of the `screen` 🙏🏼  


#
### /* MOBILE DESIGN */

![image](https://user-images.githubusercontent.com/63915024/216067498-ab90abe4-4ec1-4830-b64d-546921c97106.png)

# 

#
### /* DESKTOP DESIGN */

![image](https://user-images.githubusercontent.com/63915024/216067555-97596ddb-16a7-417d-99ed-78fa68b42d6d.png)

#

Reply:
@mukhammadrizooff  Hello good sir, If you want me to follow figma design and fulfill  desire of fix fit all screen of this world, that never going to happen, why? Because to recommendation  size of figma design is for modile is width 343px and height 836px 
![image](https://user-images.githubusercontent.com/16004612/216123944-31d0f5bd-100c-4027-a730-7eed494ac859.png)
and desktop is width 920px and height 1083px .
![image](https://user-images.githubusercontent.com/16004612/216124409-51dc8d78-a60a-499b-93bf-ae2806e51e6f.png)
So if kindly check all this number in your test machine I am sure you can have answer.

Reply:
@mukhammadrizooff  Can you please suggest me which tool you are using to test this page?

Reply:
@mukhammadrizooff this , I am getting on my test environment
![image](https://user-images.githubusercontent.com/16004612/216267341-629a92b8-b548-4f21-a9f9-a39537a1cc18.png)